portfolio/src/main/webapp/index.html

```diff
@@ -74,23 +74,24 @@ <h1>Life Before Quarantine</h1>
         </div>
       </div>
     </section>
-    <section id="comments" >
-      <ul id="comments-grid"></ul>
-      <form action="/data" method="POST">
-        <label for="comment-box">Drop a Comment!</label>
+    <section id="comments" class="page">
+      <h1>Let's Chat</h1>
+      <ul id="comments-grid" class="list-group"></ul>
+      <form action="/data" method="POST" onsubmit="return validateText()">
+        <label for="comment-box" id="comment-box-label">Drop a Comment!</label>
         <textarea name="text-input" id="comment-box"></textarea>
-        <label for="limit-comments">Number of Comments</label>
+        <input type="submit" class="btn-success">
         <select id="limit-comments" onchange="getComments()">
+          <option value="1000" disabled selected>Comments per View</option>
```

Comment:
`value="1000" ` is not needed here since the value doesn't make sense and I think you can just remove it.

Reply:
yeah, a blank string is good, and you can handle the default with that blank value explicitly rather than tricking the system into thinking the user chose 1000 from the list.